package/yast2-network.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Thu Jan 23 09:16:03 UTC 2020 - Imobach Gonzalez Sosa <igonzalezsosa@suse.com>
+
+- Do not try to read an ifcfg-* file that does not exist
+  (bsc#1161234).
```

Comment:
Any reason to not bump the version?

Reply:
Yes, it is a change that has a minimal impact, so I would prefer it to be released with the next version.